qiskit/providers/ibmq/experiment/analysis_result.py

```diff
@@ -72,6 +73,7 @@ def __init__(
             tags: Tags for this result.
             result_uuid: Unique identifier for the result.
             backend_name: Name of the backend on which the experiment was run.
+            verified: Identifies a result whose quality has been verified by a human.
```

Comment:
```suggestion
            verified: Indicates whether this result has been verified.
```
Nit - making this more generic since there are no rules enforcing it being verified by a human.

Reply:
updated the various places this description showed up